sources/burp/conf.json

```diff
@@ -0,0 +1,20 @@
+{
+    "proxy":{
+        "request_listeners":[
+            {
+                "certificate_mode":"per_host",
```

Comment:
Where do we specify the path of the CA to use ? There is two PATH in the setup script but I can't where / when they are generated and used by burp.

Reply:
I added comments on every steps of the function, you can check them to have more details and then remove those you find useless.